src/JSONFileReader.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "Board.hpp"
+
+class JSONFileReader
+{
+    std::string path_;
```

Comment:
Perhaps path_ member is not neccessary, what do you think?

Reply:
Yes, it can be removed. It's needed only in a constructor.